### PR TITLE
(maint) Download build repo from artifactory for bolt-server-runtime

### PR DESCRIPTION
**Pull request by donoghuc**
This commit updates bolt-server to update repodata from artifactory instead of enterprise.d.p.n.

### CODE REVIEW

**Comment by donoghuc** [comment]
Seems to work locally for 
```cas@cas-ThinkPad-T460p:~/working_dir/puppet-runtime$ bundle exec build pe-bolt-server-runtime-master el-8-x86_64^C
cas@cas-ThinkPad-T460p:~/working_dir/puppet-runtime$ bundle exec build pe-bolt-server-runtime-master ubuntu-18.04-amd64
```

File: configs/components/runtime-pe-bolt-server.rb [hunk 1]
```diff
@@ -8,6 +8,15 @@
     platform.provision_with "echo 'Acquire::AllowInsecureRepositories \"true\";' > /etc/apt/apt.conf.d/90insecure"
   end
 
-  platform.add_build_repository("http://enterprise.delivery.puppetlabs.net/#{settings[:pe_version]}/repos/#{platform.name}/#{platform.name}.repo")
+  artifactory_url = 'https://artifactory.delivery.puppetlabs.net/artifactory'
+
+  if platform.is_rpm?
+    platform.add_build_repository "#{artifactory_url}/rpm_enterprise__local/#{settings[:pe_version]}/repos/#{platform.name}/#{platform.name}.repo"
```
Comment (lucywyman) [inline configs/components/runtime-pe-bolt-server.rb]: It looks like this doesn't exist, at least for https://artifactory.delivery.puppetlabs.net/artifactory/rpm_enterprise__local/2019.2/repos/el-8-x86_64/ ...

Comment (donoghuc) [reply configs/components/runtime-pe-bolt-server.rb]: @puppetlabs/skeletor do we need to continue to build ofr 2019.2?

Comment (donoghuc) [reply configs/components/runtime-pe-bolt-server.rb]: Yeah i dont think we want to build that. I need to do some cleanup for 2019.2. here is a start, https://github.com/puppetlabs/ci-job-configs/pull/6841 but there is more.

**Comment by donoghuc** [comment]
Yeah, going to get the cleanups in RN> https://tickets.puppetlabs.com/browse/PE-28482

**Comment by donoghuc** [comment]
OK i think in combination with https://github.com/puppetlabs/ci-job-configs/pull/6841/commits/33ae5a82b3209d446e3f41fb03afd5453047138d (which should be merged first) this should be all the cleanup for this.

**Pull request merged**